.github/ISSUE_TEMPLATES/new-goal.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: "🧭 New team goal"
+about: Creating a new team goal
+labels: goal
+---
+
+# Background
+
+<A one to two sentence description of the goal and other background as-needed. Try to keep it short!>
```

Comment:
I wanted to use these instructions to open a goal issue and noticed that the comments between `<>` do show up in the issue preview if they contain special characters. I don't know exactly which ones, but if I remove the `!`, the comment gets hidden (this is the desired behavior, right?). Seems to be ok with having `.` though.

![goal-template](https://user-images.githubusercontent.com/7579677/108706082-c765cd00-7516-11eb-9aa5-14b1efcb7ee6.png)